README.md

```diff
@@ -1,4 +1,10 @@
-[![Build Status](https://travis-ci.org/aws/aws-xray-sdk-java.svg?branch=master)](https://travis-ci.org/aws/aws-xray-sdk-java)
+![Build Status](https://github.com/aws/aws-xray-sdk-java/actions/workflows/master-build.yml/badge.svg)
+
+### :mega: OpenTelemetry Java with AWS X-Ray
```

Comment:
Can we use the Heading 1 Casing?

```suggestion
# :mega: OpenTelemetry Java with AWS X-Ray
```

Reply:
I’m okay with that, but my preference would have been to have it Heading 1 style. I’ll defer on that decision!